deploy/helm/charts/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
 name: openebs-ndm
 description: Kubernetes Storage Device Management. Also used to maintain the inventory of block devices.
```

Comment:
n_: Change this description to the following, as it gets added to the chart release that is made. 


`Helm chart for OpenEBS Node Disk Manager - a Kubernetes native storage device management solution. For instructions on how to install, refer to  https://openebs.github.io/node-disk-manager/.`

Reply:
Done.